app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -331,7 +333,7 @@ object NoneChoice {
 
 case class Choice(
   `type`: ChoiceType,
-  options: NonEmptyList[OptionData],
+  options: List[OptionData],
```

Comment:
What will happen from user point of view if all choice are hidden?

Reply:
It will display only the title of the component so it's a broken use case. Do you have any suggestions about it?

Reply:
That's what I'd expect. We need at least on option for the choice so we should put `NonEmptyList` back. And in case we can't get a `NonEmptyList` we will let it crash stating exactly the reason what happened in the exception.